Makefile

```
@@ -333,47 +313,143 @@ test-regression: #HELP Run regression test
+test-e2e: E2E_SOURCE_MANIFEST := $(STANDARD_E2E_MANIFEST)
+test-e2e: E2E_RELEASE_MANIFEST := $(STANDARD_RELEASE_MANIFEST)
 test-experimental-e2e: GO_BUILD_EXTRA_FLAGS := -cover
-test-experimental-e2e: COVERAGE_NAME := experimental-e2e
-test-experimental-e2e: export MANIFEST := $(EXPERIMENTAL_RELEASE_MANIFEST)
-test-experimental-e2e: export INSTALL_DEFAULT_CATALOGS := false
-test-experimental-e2e: PROMETHEUS_VALUES := testdata/prometheus/values-experimental.yaml
+test-experimental-e2e: KIND_CONFIG := ./kind-config/kind-config-2node.yaml
+test-experimental-e2e: E2E_SOURCE_MANIFEST := $(EXPERIMENTAL_E2E_MANIFEST)
+test-experimental-e2e: E2E_RELEASE_MANIFEST := $(EXPERIMENTAL_RELEASE_MANIFEST)
+test-experimental-e2e: E2E_PROMETHEUS_VALUES := testdata/prometheus/values-experimental.yaml
 test-experimental-e2e: E2E_TIMEOUT ?= 25m
```

checkmake (0.3.2)

[warning] Target "test-experimental-e2e" defined multiple times (lines 334 and 339).

(uniquetargets)